homeassistant/components/screenlogic/number.py

```diff
@@ -36,6 +43,45 @@ class ScreenLogicNumberDescription(
     """Describes a ScreenLogic number entity."""
 
 
+@dataclass(frozen=True)
+class ScreenLogicPushNumberDescription(
+    ScreenLogicNumberDescription,
+    ScreenLogicPushEntityDescription,
+):
+    """Describes a ScreenLogic push number entity."""
```

Comment:
Please change these to `kw_only` and drop the mixin

Reply:
https://developers.home-assistant.io/blog/2023/12/11/entity-description-changes/